### PR TITLE
Grammar refactoring

### PR DESCRIPTION
**Pull request by RockFercho**

### CODE REVIEW

**Comment by RockFercho** [comment]
@marcocdlv @JimmyRojas @ziur please, check the changes in the project (Grammar tests transferred to the parser)

File: apex-squid/src/test/java/org/fundacionjala/enforce/sonarqube/apex/parser/grammar/ApexGrammarParameter.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+import org.junit.Test;
+
+import static org.sonar.sslr.tests.Assertions.assertThat;
+
+import static org.fundacionjala.enforce.sonarqube.apex.api.grammar.RuleKey.PARAMETER_LIST;
+import org.fundacionjala.enforce.sonarqube.apex.parser.ApexRuleTest;
+import org.junit.Before;
+
+public class ApexGrammarParameter extends ApexRuleTest {
```
Comment (JimmyRojas) [inline apex-squid/src/test/java/org/fundacionjala/enforce/sonarqube/apex/parser/grammar/ApexGrammarParameter.java]: by convention you were adding a post-fix test in the class name

ApexGrammarParameter

ApexGrammarParameterTest

**Comment by JimmyRojas** [comment]
Just a comment related to naming convention.. :)

**Comment by RockFercho** [comment]
@JimmyRojas @marcocdlv I corrected the observation

**Comment by JimmyRojas** [comment]
looks good to me

**Pull request merged**